redisinsight/ui/src/pages/home/components/CloudConnection/CloudConnectionForm/CloudConnectionForm.tsx

```diff
@@ -16,7 +16,6 @@ import {
   keys,
 } from '@elastic/eui'
 
-import RESOURCES from 'uiSrc/resourses/en-EN'
 import { validateField } from 'uiSrc/utils/validations'
```

Comment:
Can you check RESOURCES, i guess it is don't need anymore. Probably better remove it from resources

Reply:
I wouldn't delete it. next time if we will need it we have to create everything from the scratch